tests/agent/test_prompt.py

```
@@ -0,0 +1,250 @@
+
+        # Assert both tests passed
+        assert small_passed and large_passed, "Kernel verification tests failed"
+
+    except Exception as e:
+        print(f"Compilation failed: {e}")
+        save_test_results(output_file, model_name, cuda_code, False, str(e))
+        pytest.fail(f"Compilation failed: {e}")
+
+
+if __name__ == "__main__":
+    pytest.main([__file__] + sys.argv[1:])
```

Ruff (0.14.10)

Consider `[__file__, *sys.argv[1:]]` instead of concatenation

Replace with `[__file__, *sys.argv[1:]]`

(RUF005)